examples/api-extension/runner/src/main.rs

```diff
@@ -0,0 +1,101 @@
+/* Copyright (c) Fortanix, Inc.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+extern crate aesm_client;
+extern crate enclave_runner;
+extern crate sgxs_loaders;
+extern crate failure;
+#[macro_use]
+extern crate clap;
```

Comment:
Don't use clap, trim the example code to have the minimal code for what you want to demonstrate

Reply:
Removed dependency on Clap and Failure.